README.md

```diff
@@ -90,9 +90,9 @@ Replace the paths with the appropriate paths within the `backend` repo as approp
 
 #### Installing the trunk launcher
 
-This project uses [trunk](https://trunk.io) to lint and format its code.
+This project uses [trunk](https://trunk.io) to lint and format its code. Trunk is installed as a dev dependency so you can invoke it as `npx trunk`.
 
-You can [install](https://docs.trunk.io/getting-started) the trunk launcher using the command `curl https://get.trunk.io -fsSL | bash`. This will place `trunk` in your PATH.
+You can alternatively [install](https://docs.trunk.io/getting-started) the trunk launcher using the command `curl https://get.trunk.io -fsSL | bash`. This will place `trunk` in your PATH. Or you can run `npm install -g @trunkio/launcher` to install globally and allow `npm` to manage your `trunk` launcher installation.
```

Comment:
Good call - the docs are no longer needed since the tool will be installed over npm.

Reply:
sorry for dropping off here- would you like me to remove this section entirely? Given trunk is so new it might be nice to keep a link to the docs, but makes sense to remove the installation instructions. Let me know what you think.

Reply:
Docs are updated - please take a look!